libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/InsightAlertConfig/EditAlert.tsx

```diff
@@ -139,6 +156,21 @@ export const EditAlert: React.FC<IEditAlertProps> = ({
                             measures={measures}
                             overlayPositionType={overlayPositionType}
                         />
+                        {Boolean(canManageAttributes && attributes.length > 0) && (
```

Comment:
Boolean conversion seems unnecessary?

Reply:
Eslin is complaining about Potentinally leak value :D SO i fix it like was explained in example